code/datums/quirks/negative_quirks/negative_quirks.dm

```diff
@@ -741,14 +743,22 @@
 	if(!length(organ_slots)) //what the hell
 		return
 	for(var/organ_slot in possible_organ_slots)
+		old_organ = human_holder.get_organ_slot(organ_slot)
```

Comment:
This doesn't work, the quirk makes up to 4 organs and you can't store 4 vars in 1 var directly (it's not a list)

You would have to make it a list, but also I don't see that as necessary

Call `Insert` with `drop_if_replaced = FALSE` instead, and in `remove` just delete all the crappy one and call regenerate organs